nemo/collections/common/parts/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nemo.collections.common.parts.multi_layer_perceptron import MultiLayerPerceptron
```

Comment:
It seems to me that MultiLayerPerceptron should be under "nemo.collections.common.modules.MultiLayerPerceptron"

Reply:
It's a part of a module, I'd prefer to leave it under parts

Reply:
in this case you should inherit it from torch.nn.Module, not pl.LightningModule. Is there a reason you used LightningModule ?

Reply:
it actually inherits from torch.nn.Module already, git hub doesn't show the latest https://github.com/NVIDIA/NeMo/blob/63017e6c531c75b9ffe4d48bdca63f50f569d65a/nemo/collections/common/parts/multi_layer_perceptron.py#L18